ngraph/frontend/onnx_common/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "onnx_common")
```

Comment:
Please do not use `TARGET_NAME` variable its value should not change and it make searching more dificult.

Reply:
I see your arguments (especially because we don't change target name very often), but it is convention from other OV cmake files.
@ilyachur Have you maybe some other arguments why we use such convention?

Reply:
I don't fully understand the issue.
In other projects you always need to use `onnx_importer`. You use `TARGET_NAME` only inside the local cmake file.
And I don't see any issues with search, because search also will contain the line `set(TARGET_NAME onnx_importer)` and you will be able to find local cmake script where you use `TARGET_NAME` for onnx importer.